java/src/test/java/org/taymyr/lagom/elasticsearch/search/dsl/query/compound/CompoundTest.java

```diff
@@ -59,4 +61,52 @@ void shouldSuccessfullySerializeFilteredBoolQuery() {
         assertThatJson(actual).isEqualTo(expected);
     }
 
+    @Test
```

Comment:
Merge 4 test to one with all properties `must`, `mustNot`, `should`, `filter`.

Reply:
Merged